<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meetingClientClass</class>
 <widget class="QMainWindow" name="meetingClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>meetingClient</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{color: #555;font-weight:bold;}
QPushButton::hover{border:none;background-color:#AAA;color: #FFF;font-weight:bold;}
QPushButton::pressed{border:none;color: #CCC;}
QLbel{color: #AAA;}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="grpSever">
        <property name="title">
         <string>连接服务器</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labSeverIp">
           <property name="text">
            <string>ip：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineSeverIp">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>172.30.126.241</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labSeverPort">
           <property name="text">
            <string>port：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineSeverPort">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>8888</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnConnect">
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpRoom">
        <property name="title">
         <string>会议室</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labRoomId">
           <property name="text">
            <string>会议室编号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineRoomId">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRoomJoin">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>加入</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRoomCreate">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>创建</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRoomExit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="openVideo">
        <property name="text">
         <string>相机</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="openAudio">
        <property name="text">
         <string>语音</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTest">
        <property name="text">
         <string>测试</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="listTextMsg">
          <property name="minimumSize">
           <size>
            <width>512</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>512</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineSendMsg">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>512</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>在此输入需要发送的信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSendMsg">
          <property name="maximumSize">
           <size>
            <width>512</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labVideo">
        <property name="minimumSize">
         <size>
          <width>512</width>
          <height>512</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>512</width>
          <height>512</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: gray;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listVideo">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1333</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{color: #555;font-weight:bold;}
QPushButton::hover{border:none;background-color:#AAA;color: #FFF;font-weight:bold;}
QPushButton::pressed{border:none;color: #CCC;}
QLbel{color: #AAA;}</string>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="meetingClient.qrc"/>
 </resources>
 <connections/>
</ui>
